src/rpc/core_rpc_server.cpp

```diff
@@ -3570,6 +3570,93 @@ namespace cryptonote
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool core_rpc_server::on_get_height_by_time(const COMMAND_RPC_GET_HEIGHT_BY_TIME::request& req, COMMAND_RPC_GET_HEIGHT_BY_TIME::response& res, epee::json_rpc::error& error_resp, const connection_context *ctx)
```

Comment:
Might want to also follow up with a request to add this method to the online developer documentation if it gets merged. See [this](https://www.getmonero.org/resources/developer-guides/daemon-rpc.html).

Reply:
Yes, this should be done if this PR ever gets merged.